examples/asyncio_displayio_button.py

```diff
@@ -0,0 +1,332 @@
+# SPDX-FileCopyrightText: 2022 Tim Cocks for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+"""
+Example that illustrates how to use Displayio Buttons to modify
+some blinking circles. One button inverts colors, the others change
+the interval length of the blink for one of the circles.
+"""
+
+import asyncio
+import adafruit_touchscreen
+import displayio
+import terminalio
+import vectorio
+import board
+from adafruit_button import Button
+
+# use built-in display
+display = board.DISPLAY
+
```

Comment:
Needed a `display.rotation = 0` here or the touch position didn't match the display after being changed in a different piece of code.

Reply:
Ordinarily the display will default to `0` rotation but since the display stays alive after the circuitpython VM instance is gone or restarts it would keep it's rotation if it did get changed.

It would be good practice to include the explict code to set it back to `0` just in case it has been changed in the users environment. I'll add a commit now. with this change. Thanks for the suggestion.